dist/es-module-shims.js

```diff
@@ -689,6 +693,32 @@
     throw new Error();
   }
 
+  class WorkerShim {
+    constructor(aURL, options = {type: 'classic'}) {
+      if (options.type !== 'module') {
+        return new Worker(aURL, options);
+      }
+
+      let es_module_shims_src = new URL('es-module-shims.js', baseUrl).href;
+      const scripts = document.scripts;
+      for (let i = 0, len = scripts.length; i < len; i++) {
+        if (scripts[i].src.includes('es-module-shims.js')) {
+            es_module_shims_src = scripts[i].src;
+
+            break;
+        }
+      }
+
+      const workerScriptUrl = createBlob(`importScripts('${es_module_shims_src}'); 
+    self.importMap = ${JSON.stringify(options.importMap || {})}; importShim('${new URL(aURL, baseUrl).href}')`);
```

Comment:
Indentation.

Reply:
I think I submitted by mistake this file, namely `dist/es-module-shims.js`. I shouldn't have done that. I apologize.